### ISSUE TITLE
come up with a few 'killer' conservation focused apps

### ISSUE THREAD

**Issue by lzachmann** [csp-inc/landcover#4]
We were thinking about solar panels. Can we think about what the data requirements would be? (E.g., NAIP before and after?) And what about wildfires, or oil and gas or something too?

**Comment by blindjesse** [comment]
Well I think ultimately we were thinking _all_ development, solar panels were just a possible first. I was trying to think of things that change fast enough to have a before and after. Watching the presentation last week, I'm not sure we even need a before and after for all areas (I guess we would for demonstration purposes). Could just train on the areas with photos of panels that match our training data and classify on whatever we want?

**Comment by lzachmann** [comment]
Yes I think so, even using the current app, though displaying the before / after directly will likely require new frontend features. As it stands, I think you can bring in one basemap, retrain, save checkpoint, then bring in another and classify using the same model checkpoint. But yeah, no way to quickly toggle between the two so far as I know.
